src/assessment.js

```diff
@@ -58,5 +65,7 @@ var q4 = "/* your answer here */";
 // the context of `alderaan`.
 //
 var q5 = function fireOnAlderaan () {
-  // your code here
+  deathStar.fire(alderaan);
```

Comment:
No, `fire` doesn't take any arguments. How do we set the context (i.e. the meaning of `this`) for the `fire` function?